### ISSUE TITLE
8.0.0 is missing some of the test files

### ISSUE THREAD

**Issue by mgorny** [pikepdf/pikepdf#490]
```
Only in /tmp/mgorny-dev-scripts/portage/dev-python/pikepdf-7.2.0/work/pikepdf-7.2.0/tests: conftest.py
Only in /tmp/mgorny-dev-scripts/portage/dev-python/pikepdf-7.2.0/work/pikepdf-7.2.0/tests: resources
```

This makes it impossible to run the test suite against sdist.

**Comment by jbarlow83** [comment]
Fixed in 8.1.0

**Comment by mgorny** [comment]
Thank you.
